test/any/parsing/WebKitParser-test.js

```diff
@@ -49,4 +49,22 @@ describe('WebKitParser', function() {
     `);
     expect(p.value).toBeDefined();
   });
+  it('should report parse complete status to console', function() {
```

Comment:
Don't need a second set of `logParse()` tests once we move them to `BaseParser-test.js`.